commands/host/setup

```diff
@@ -49,7 +49,7 @@ handle_options() {
   while [ $# -gt 0 ]; do
     case $1 in
       -a | --all)
-        mapfile -t SITES < <(drush site:alias --fields=name --format=list --filter=.local | sed 's/@//' | sed 's/.local//')
+        get_sites
```

Comment:
I like the idea, but I would rename the get_sites function to something more like ‘set_sites_variable’ or something like that. At least to know that this is generating a global SITES variable.